include/deal.II/fe/mapping_fe_field.h

```diff
@@ -563,49 +590,78 @@ class MappingFE : public Mapping<dim,spacedim>
   void update_internal_dofs(const typename Triangulation<dim,spacedim>::cell_iterator &cell) const;
 
 
+  /** Reimplemented from Mapping. See the documentation of the base class for
+   * detailed information.
+   */
   mutable std::vector<double> local_dofs;
```

Comment:
I don't understand this. If you reimplement  a function,that makes sense. But why would you not simply use the variable from the base class?